spec/xcpretty/parser_spec.rb

```diff
@@ -321,6 +321,10 @@ module XCPretty
       @parser.parse(SAMPLE_SPECTA_SUITE_BEGINNING)
     end
 
+    it "parses huge failure messages in a timely manner" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.